src/components/SignUpForm/styles.scss

```diff
@@ -0,0 +1,37 @@
+@import "../../styles/main.scss";
+
+.container {
+    min-height: 100vh;
+    @include flex-center();
+}
+
+.signUp-form {
+    margin: auto;
+    height: rem(400px);
+    width: rem(650px); 
+    flex-direction: column;
+    
+    background-color: $background;
+  
+    @include flex-center();
+}
+
+@mixin button-bg($background) {
+  background-color: $background;
+  margin-top: 16px;
+  &:hover {
+    background-color: darken($background, 8%);
+  }
+}
```

Comment:
If you are really sure to make a reusable `@mixin`, then please move this to `styles/_mixins.scss` for best practice.